rero_ils/modules/imports/api.py

```diff
@@ -375,11 +378,13 @@ def _split_stream(stream):
                             }
                             self.data.append(json_data)
                             self.results['hits']['hits'].append(data)
-                    self.results['hits']['remote_total'] = int(etree.parse(
-                            BytesIO(response.content))
-                            .find('{*}numberOfRecords').text)
+                    self.results['hits']['remote_total'] = 0
+                    with contextlib.suppress(Exception):
+                        self.results['hits']['remote_total'] = int(etree.parse(
+                                BytesIO(response.content))
+                                .find('{*}numberOfRecords').text)
```

Comment:
I think it's not the pythonic way to use. `contextlib.suppress(Exception)` should be used only as last option.
In this case, a simple `try...except` seems better :
```python
try:
    self.results['hits']['remote_total'] = int(etree.parse(
          BytesIO(response.content))
          .find('{*}numberOfRecords').text)
except Exception:
   self.results['hits']['remote_total'] = 0
```